src/main/java/org/cactoos/scalar/ScalarWithFallback.java

```diff
@@ -91,18 +92,20 @@ private T fallback(final Throwable exp) throws Exception {
         final Sorted<Map.Entry<FallbackFrom<T>, Integer>> candidates =
             new Sorted<>(
                 Comparator.comparing(Map.Entry::getValue),
-                new Filtered<>(
-                    entry -> new Not(
-                        new Equals<>(
-                            entry::getValue,
-                            () -> Integer.MIN_VALUE
-                        )
-                    ).value(),
-                    new MapOf<>(
-                        fbk -> fbk,
-                        fbk -> fbk.support(exp.getClass()),
-                        this.fallbacks
-                    ).entrySet().iterator()
+                new IterableOf<>(
+                    new Filtered<>(
```

Comment:
@mazdack directly use the `Filtered` and `Sorted` class from `org.cactoos.iterable`